background.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
this isn't code you wrote right?

Reply:
It's mostly code I copied, but I added in a few lines. Was not sure what to do about the copyright.